scss/detail.scss

```diff
@@ -0,0 +1,79 @@
+@import 'colors';
```

Comment:
You can import the colors in the `application.scss` so you won't have to do it in every sass file you need those variables